src/unix/fs.c

```diff
@@ -525,7 +525,7 @@ static ssize_t uv__fs_pathmax_size(const char* path) {
   pathmax = pathconf(path, _PC_PATH_MAX);
```

Comment:
Glibc does these in the other order (prefers to use the header value, and falls back to this call if that's undefined)
https://github.com/lattera/glibc/blob/master/stdlib/canonicalize.c#L68-L74. Perhaps worth doing in libuv that way also? IIUC, the `pathconf` value is defined to be strictly larger than the macro value and is noted in its documentation that it thus may be unsuitable for allocating memory.